book-service/database.js

```diff
@@ -0,0 +1,47 @@
+import mysql from 'mysql2';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const pool = mysql.createPool({
+  host: process.env.MYSQL_HOST,
+  user: process.env.MYSQL_USER,
+  password: process.env.MYSQL_PASSWORD,
+  database: process.env.MYSQL_DATABASE
+}).promise();
+
+export async function getPublishers() {
+  const [result] = await pool.query("SELECT * FROM publishers")
```

Comment:
```suggestion
  const [result] = await pool.query("SELECT * FROM publishers");
```
i think you forgot this file haha